### PR TITLE
Suities (Adds Vanax species)

### PR DESCRIPTION
**Pull request by elgeonmb**
Adds the Vanax, a race of suitbound bug-people

(Edited by Ater to actually indicate what this changes, because it took me several minutes to figure that out while looking at the diff)

### CODE REVIEW

**Comment by BillyBangles** [comment]
bugmen hype

File: code/modules/mob/living/carbon/human/species/station/station.dm [hunk 1]
```diff
@@ -287,6 +287,82 @@
...
+	warning_high_pressure = WARNING_HIGH_PRESSURE + 500 // High pressure warning. Placeholderish
+	warning_low_pressure = 300   // Low pressure warning. Placeholder
+	hazard_low_pressure = 220     // Dangerously low pressure.
+	safe_pressure = 400
+	poison_type = "nitrogen"                        // technically it's a partial pressure thing but IDK if we can emulate that
+
+	genders = list(FEMALE, NEUTER)
+
+
+	spawn_flags = SPECIES_CAN_JOIN// | SPECIES_IS_WHITELISTED
```
Comment (Atermonera) [inline code/modules/mob/living/carbon/human/species/station/station.dm]: Supposed to be commented, or was that just for testing?

File: code/modules/mob/living/carbon/human/species/station/station.dm [hunk 2]
```diff
@@ -287,6 +287,82 @@
...
+	num_alternate_languages = 3
+	secondary_langs = list(LANGUAGE_VANAX, LANGUAGE_UNATHI)
+	name_language = LANGUAGE_VANAX
+	species_language = LANGUAGE_VANAX
+	health_hud_intensity = 2.5
+
+	min_age = 16
+	max_age = 90
+
+	blurb = "THEY BE FUCKIN BUG MEN WHAT LIVE IN LIL SUITS. GODDAMN LOVE THEM."
```
Comment (Atermonera) [inline code/modules/mob/living/carbon/human/species/station/station.dm]: Pls

File: code/stylesheet.dm [hunk 3]
```diff
@@ -98,6 +98,7 @@ h1.alert, h2.alert		{color: #000000;}
 .solcom					{color: #22228B;}
 .changeling				{color: #800080;}
 .vox					{color: #AA00AA;}
+.vanax					{color: #941C1C;}
```
Comment (Atermonera) [inline code/stylesheet.dm]: The color For reference. Might be a touch close to sec?

**Comment by Atermonera** [comment]
Also, because it's relevant, this is a pinned message in the Lore channel on discord (The bottom one):
message

**Comment by elgeonmb** [comment]
HEY THIS WASN'T SUPPOSED TO HAPPEN

**Comment by BillyBangles** [comment]
press f to pay respects

**Comment by Belsima** [comment]
big regret

**Pull request closed without merging**